src/dashboards/teachers/common.libsonnet

```
@@ -0,0 +1,116 @@
+// Video commons
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local template = grafana.template;
+local common = import '../common.libsonnet';
+
+{
+  datasources: {
+    lrs: 'lrs',
+    edx: 'edx',
```

```suggestion
    edx: 'edx_app',
```
Isn't this the same as in the `./etc/grafana/provisioning/datasources/mysql.yaml` ? Or maybe change to `edx` in `mysql.yaml` ?